pkg/subnet/local.go

```diff
@@ -217,6 +229,10 @@ func (d *LocalDeployer) doDeploy(chain string, chainGenesis []byte, genesisPath
 
 	clusterInfo, err = WaitForHealthy(ctx, cli)
 	if err != nil {
+		secondErr := d.removeInstalledPlugin(chainVMID)
```

Comment:
(note this is related to LoadSidecard, not to this line). can we move LoadSidecard errors to before BlockchainCreation? Probably would like all the errors after CreateBlockchain, to cause plugin removal

Reply:
moved it